Create ui-c8y-1021-77-1-the-unit-should-not-be-relevant-when-exporting-data-9133-graft.md (#3406)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-77-1-the-unit-should-not-be-relevant-when-exporting-data-9133-graft.md

---
date: ""

> title: The measurements without units could not be exported correctly

Reviewer: ```suggestion
title: Measurements are now exported correctly in data exports
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63706
version: 1021.77.1
---

> Previously, measurements without unit resulted in exports containing only headers while omitting the actual measurement values. This bug has been fixed, ensuring complete data export including both headers and values.

Reviewer: ```suggestion
Previously, when exporting data, measurements without units contained only headers while omitting the actual measurement values. This issue has been fixed, ensuring complete data export including both headers and values.
```

---
date: ""

> title: The unit should not be relevant when exporting data (#9133) [GRAFT][release/cd] (#9276)

Reviewer 2: The Claude AI suggested to use the following title:
```suggestion
title: The measurements without units could not be exported correctly
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63706
version: 1021.77.1
---

> Co-authored-by: Lukasz Janusz <114157358+luja-c8y@users.noreply.github.com>
> Co-authored-by: Lukasz Janusz <lukasz.janusz@cumuloicty.com>

Reviewer 2: The Claude AI suggested to use the following body:
```suggestion
Previously, measurements without unit resulted in exports containing only headers while omitting the actual measurement values. This bug has been fixed, ensuring complete data export including both headers and values.
```
